.github/workflows/iac-validate.yml

```
@@ -0,0 +1,30 @@
+name: IAC Validation
```

Let's be consistent with casing
```suggestion
name: IaC Validation
```